content/user/vulns/ManageVulns.md

```diff
@@ -26,6 +26,32 @@ Contrast provides multiple ways to narrow findings and focus on the vulnerabilit
 
 <a href="assets/images/Vulnerability_Filters.png" rel="lightbox" title="Vulnerability Filters"><img class="thumbnail" src="assets/images/Vulnerability_Filters.png"/></a>
 
+## Pending Vulnerabilities 
+
+If an administrator [requires administrative approval](admin-orgsettings.html#app-defaults) to close vulnerabilities, vulnerabilities with their specified statuses and severities go into a **Pending** state when you attempt to close them. Administrative approval applies to changes to vulnerabilities by two-way [bugtracker integrations](admin-orgintegrations.html#bugtracker) as well as [auto-remediation policies](admin-policymgmt.html#remediate).  
```

Comment:
Suggestions:

"...vulnerabilities with **the configured** statuses and severities go into..." (or maybe just "**the** specified...")

"Administrative approval applies to changes **in** vulnerabilities by two-way...